src/device-registry/controllers/create-event.js

```diff
@@ -122,3 +124,99 @@ const createEvent = {
 };
 
 module.exports = createEvent;
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
Please shed some light on this commented out code, what does this change mean? @NoahNsimbe